tracer/src/Datadog.Trace.Coverage.collector/Datadog.Trace.Coverage.collector.csproj

```diff
@@ -1,7 +1,6 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
     <PropertyGroup>
-        <TargetFramework>netstandard2.0</TargetFramework>
```

Comment:
I don't think we want to remove this, otherwise we're building for all the other TFMs etc. I suspect using `<TargetFrameworks>` _might_ fix it 🤔